packages/labs/test/timezonePickerTests.tsx

```diff
@@ -0,0 +1,303 @@
+        const placeholder = "test placeholder";
+        const timezonePicker = shallow(
+            <TimezonePicker popoverProps={{ isOpen: true, inline: true }} inputProps={{ placeholder }} />,
+        );
+        const inputGroup = findInputGroup(timezonePicker);
+        assert.strictEqual(inputGroup.prop("placeholder"), placeholder);
+    });
+
+    it("shows initial items when the query is empty", () => {
+        let select: SelectShallowWrapper;
+        const date = new Date();
+        const timezonePicker = shallow(<TimezonePicker date={date} popoverProps={{ isOpen: true, inline: true }} />);
+        select = findSelect(timezonePicker);
+        const items = select.prop("items");
+        assert.deepEqual(items, getInitialTimezoneItems(date, true));
+    });
+
+    it("shows all items when the query is non-empty", () => {
+        const date = new Date();
+        const query = "test query";
+        const timezonePicker = shallow(
+            <TimezonePicker date={date} popoverProps={{ isOpen: true, inline: true }} inputProps={{ value: query }} />,
+        );
+        assert.strictEqual(timezonePicker.state("query"), query);
+        const select = findSelect(timezonePicker);
+        const items = select.prop("items");
+        assert.deepEqual(items, getTimezoneItems(date));
+    });
+
+    it("uses the same items between non-empty query changes", () => {
+        let select: SelectShallowWrapper;
+        let inputGroup: InputGroupShallowWrapper;
+
+        const timezonePicker = shallow(<TimezonePicker popoverProps={{ isOpen: true, inline: true }} />);
+
+        inputGroup = findInputGroup(timezonePicker);
+        const query1 = "test query 1";
+        inputGroup.simulate("change", { currentTarget: { value: query1 } });
+        assert.strictEqual(timezonePicker.state("query"), query1);
+
+        select = findSelect(timezonePicker);
+        const items1 = select.prop("items");
+
+        inputGroup = findInputGroup(timezonePicker);
+        const query2 = "test query 2";
+        inputGroup.simulate("change", { currentTarget: { value: query2 } });
+        assert.strictEqual(timezonePicker.state("query"), query2);
+
+        select = findSelect(timezonePicker);
+        const items2 = select.prop("items");
+
+        assert.strictEqual(items1, items2);
+    });
+
+    it("renders local timezone at top of item list if show local timezone is enabled", () => {
+        const timezonePicker = shallow(
+            <TimezonePicker popoverProps={{ isOpen: true, inline: true }} showLocalTimezone={true} />,
+        );
+        const select = findSelect(timezonePicker);
+        const items = select.prop("items");
+        assert.isTrue(items.length > 0);
+        const firstItem = items[0];
+        assert.strictEqual(firstItem.timezone, moment.tz.guess());
+    });
+
+    it("does not render local timezone at top of item list if show local timezone is disabled", () => {
```

Comment:
use prop name in test name: `if showLocalTimezone=false` 
this comment applies widely to this file.